modules/microshift-accessing-cluster-remotely-non-admin.adoc

```diff
@@ -2,15 +2,13 @@
 //
 // microshift/microshift-install-rpm.adoc
 
-:_content-type: CONCEPT
-[id="accessing-microshift-cluster-remotely_{context}"]
+:_content-type: PROCEDURE
+[id="accessing-microshift-cluster-remotely-non-admin_{context}"]
 = Accessing the {product-title} cluster remotely
 
-Use the following procedure to access the {product-title} cluster from a remote workstation. The procedure assumes that there are two users:
+Use the following procedure to access the {product-title} cluster from a remote workstation.
 
-* The first user, `user@microshift`, is the user on the {product-title} host machine and is responsible for setting up the `op-system` machine so that it can be accessed by a remote user on a separate workstation.
-
-* The second user, `user@workstation`, accesses the {op-system} machine remotely from their workstation. The `<user>` value in the procedure is the name of the user that `user@workstation` uses to log in to the {product-title} host.
+* `user@workstation`, accesses the host machine remotely. The `<user>` value in the procedure is the name of the user that `user@workstation` logs in with to the {product-title} host.
```

Comment:
We cannot split this procedure into two:
In order for `user@workstation` (who doesn't need admin rights) to remotely access the cluster running on the `microshift` host, the `user@microshift` (who must have admin rights) has to open the firewall first.
These steps are inseparable, so should be in a single section / procedure.